code/game/objects/items/sellable.dm

```diff
@@ -0,0 +1,100 @@
+/obj/item/sellable/salvage
```

Comment:
"sellable" is necessary? From what i see nothing is inherited from it and "salvage" is the parent from all variations.

Reply:
The original intention was the file to include all generic sellable items for eventual expansion, but I can simplify it by making the file into purely a salvage file. I'll do so now.

Reply:
Done